docs_user/assemblies/assembly_migrating-the-object-storage-service.adoc

```diff
@@ -4,13 +4,14 @@ ifdef::context[:parent-context: {context}]
 
 :context: migrate-object-storage-service
 
-= Migrating the {object_storage_first_ref} to {rhos_long} nodes
-This section only applies if you are using {rhos_prev_long} {object_storage_first_ref} as Object Storage
-service. If you are using the Object Storage *API* of Ceph Object Gateway (RGW), you can skip this section.
+= Migrating the {object_storage} to {rhos_long_noacro} nodes
+If you are using the {rhos_prev_long} {object_storage_first_ref} as an Object Storage service, you must migrate your Object Storage service to {rhos_long_noacro} nodes. If you are using the Object Storage API of the Ceph Object Gateway (RGW), you can skip this chapter.
 
-Data migration to the new deployment might be a long running process that runs mostly in the background. The {object_storage} replicators will take care of moving data from old to new nodes, but depending on the amount of used storage this might take a very long time. You can still use the old nodes as long as they are running and continue with adopting other services in the meantime, reducing the amount of downtime. Note that performance might be decreased to the amount of replication traffic in the network.
+The data migration happens replica by replica. For example, if you start with 3 replicas, you only move 1 one of them at a time to ensure that the remaining 2 replicas are still available and that you can use the {object_storage} during the migration.
+
+[NOTE]
+Data migration to the new deployment is a long-running process that runs mostly in the background. The {object_storage} replicators move data from old to new nodes, but depending on the amount of used storage, this might take a long time. To reduce downtime, you can use the old nodes if they are running and continue with adopting other services in the meantime. Performance might be decreased to the amount of replication traffic in the network.
```

Comment:
1. Can we give an estimate of how long it might take to move data from old to new nodes?
2. Regarding the last sentence in the note, is that supposed to say "Performance might be degraded **due to** the mount of replication traffic in the network"?

Reply:
1. Unfortunately not - can be minutes up to months. It depends mostly on the amount of storage, but also on disk speed, network speed, overall load etc.
2. Yes